modules/_node-scripts/configuration/getProjectEntryPoints.mjs

```diff
@@ -11,25 +11,55 @@ import projectScopeRequire from '../util/projectScopeRequire.mjs';
  *
  * {
  *   main: 'src/main/resources/META-INF/resources/index.js',
- *   typescript: 'src/main/resources/META-INF/resources/index.d.ts'
+ *   submodules: {
+ *   	foo: 'src/main/resources/META-INF/resources/foo.js'
+ *   },
+ *   typescript: {
+ *   	main: 'src/main/resources/META-INF/resources/index.d.ts'
+ *   }
  * }
  */
 export default function getProjectEntryPoints(projectDir = '.') {
-	const {main, typescript} = projectScopeRequire(
+	const {main, submodules, typescript} = projectScopeRequire(
 		'./node-scripts.config.js',
 		projectDir
 	);
 
 	const entryPoints = {};
 
 	if (main) {
+		verifyResourcePath(main, 'main');
+
 		entryPoints.main = main;
 		entryPoints.typescript = main;
 	}
 
 	if (typescript && typescript.main) {
+		verifyResourcePath(typescript.main, 'typescript');
+
 		entryPoints.typescript = typescript.main;
 	}
 
+	if (submodules) {
+		Object.values(submodules).forEach((submodulePath) => {
+			verifyResourcePath(submodulePath, 'submodule');
+		});
+
+		entryPoints.submodules = submodules;
+	}
+
 	return entryPoints;
 }
+
+function verifyResourcePath(resourcePath, type) {
+	if (
+		resourcePath.startsWith('./src/main/resources/META-INF/resources') ||
+		resourcePath.startsWith('./src/node/')
```

Comment:
`src/node` ? What's that? :thinking:

Reply:
It's the generated rest-client-js modules for headless. I tried to get them to change this in the past and they didn't budge, so I just added this here to support that